README.md

```diff
@@ -1 +1,4 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+My first Mycroverse proyect
+Build in html and CSS
```

Comment:
- [ ] Please make sure the Microverse README template is used to create the README. You can find the template [here](https://github.com/microverseinc/readme-template/blob/master/README.md)